prisma/migrations/20240116163020_make_use_case_and_country_required/migration.sql

```diff
@@ -0,0 +1,10 @@
+/*
+  Warnings:
+
+  - Made the column `country` on table `users` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `use_case` on table `users` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "users" ALTER COLUMN "country" SET NOT NULL,
+ALTER COLUMN "use_case" SET NOT NULL;
```

Comment:
Since we're setting these to not allow nulls, I think we also need to update the code that creates new users so that the country and use case are provided. Otherwise, this will probably fail. We don't have E2E test coverage at the moment to catch these in CI. That's something I've been meaning to add.

Reply:
oh yeah that's true! because we aren't providing it yet. let me add that